app/services/hulu_mapping_service.rb

```diff
@@ -0,0 +1,81 @@
+# Service to sync Hulu API with Kitsu
+class HuluMappingService
+  HULU_URL = 'https://distribution.hulu.com/api/v1/'.freeze
+
+  attr_reader :since
+
+  # Initialize service with optional datetime
+  def initialize(since = nil)
+    @since = since
+  end
+
+  # Grabs all available series from hulu
+  def get_series(limit)
+    offset = 0
+    series_res = Typhoeus.get(
+      "#{HULU_URL}/series?guid=#{guid}&primary_category=animation&offset=#{offset}&limit=#{limit}"
+    )
+    return unless series_res.success?
+    JSON.parse(series_res.body)['results']
+  end
+
+  # Start of syncing series and episodes on hulu with kitsu anime
+  def sync_series_and_episodes
+    limit = 10_000
```

Comment:
Is this basically a constant?

Reply:
yeah, i can put it in the url now if you want to remove this

Reply:
Yeah I'd do that or at least move it to like `LIMIT = 10_000`